src/debugger_requests.jl

```diff
@@ -128,6 +128,15 @@ function set_compiled_functions_modules!(items::Vector{String})
 
     # user wants these compiled:
     for acc in items
+        if acc == "ALL_MODULES_EXCEPT_MAIN"
+            for mod in values(Base.loaded_modules)
```

Comment:
Question: does the loaded modules contain all submodules as well? The submodules shall also be set to ```compiled```, right?